SwipeActions.js

```diff
@@ -101,22 +101,27 @@ export default class SwipeActions extends React.Component {
   }
 
   render() {
+    let actionViewRight = 0;
     let actionViews = this.state.width === 0 ?
         null : this.props.actions.map((action, index) => {
-      return (
+      let customStyle = (action.containerStyle) ? action.containerStyle : {};
+      let actionWidth = (action.width) ? action.width : ACTION_BUTTON_DEFAULT_WIDTH;
```

Comment:
In JS people tend to write `action.width || ACTION_BUTTON_DEFAULT_WIDTH` when providing defaults for falsy values. In this case though we might want to allow 0 as a valid (but not very useful) width so can you keep the ternary expression and change the check to `action.width != null` to specifically look for null/undefined?